provision/aws/route53/route53_create_failover_lambda.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -e -o pipefail
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/route53_common.sh
+
+if [ -z "${HEALTH_CHECK_ID}" ]; then
+  echo "HEALTH_CHECK_ID variable must be set"
+  exit 1
+fi
+
+ALARM_NAME=${HEALTH_CHECK_ID}
+TOPIC_NAME=${HEALTH_CHECK_ID}
+FUNCTION_NAME=${HEALTH_CHECK_ID}
+ROUTE53_REGION="us-east-1"
+
+export AWS_REGION=${ROUTE53_REGION}
+
+TOPIC_ARN=$(aws sns create-topic --name ${TOPIC_NAME} \
+  --query "TopicArn" \
+  --tags "Key=HealthCheckId,Value=${HEALTH_CHECK_ID}" \
+  --output text
+)
+
+# Create CloudWatch alarm
+aws cloudwatch put-metric-alarm \
+  --alarm-actions ${TOPIC_ARN} \
+  --actions-enabled \
+  --alarm-name ${ALARM_NAME} \
+  --dimensions "Name=HealthCheckId,Value=${HEALTH_CHECK_ID}" \
+  --comparison-operator LessThanThreshold \
+  --evaluation-periods 1 \
+  --metric-name HealthCheckStatus \
+  --namespace AWS/Route53 \
+  --period 60 \
```

Comment:
At what stage the 60 second evaluation period starts for this metric alarm ?

Reply:
I believe it starts as soon as the alarm is created.